src/Controllers/UserController.js

```diff
@@ -134,6 +134,27 @@ export class UserController extends AdaptableController {
     });
   }
 
+  resendVerificationEmail(username, token) {
+    var where = {};
+    if (username) {
+      where.username = username;
+    }
+    if (token) {
+      where._email_verify_token = token;
+    }
```

Comment:
Yeah...I just didn't want people to be able to guess someone else's username and unverify their email by hitting that page (directly) and re-sending the link, but I guess the better solution is to not have the resend process happen on the server if the email is already verified.